policies/permissions.md

```diff
@@ -0,0 +1,102 @@
+different [roles in the project](https://www.astropy.org/team.html#roles).
+
+Occasional *temporary* write (or higher) permissions may be granted for specific
+tasks (for example, a maintainer may be given admin rights to a repo to
+configure CI for the first time or similar). Such permissions must be done
+temporarily unless prescribed otherwise by this document.
+
+For GitHub, the
+[permissions](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization)
+are enforced by adding individuals to the GitHub team
+matching their role (for example, "Astropy Core Maintainers" or "Astropy
+website maintainers" for a core maintainer or the astropy.org team, respectively),
+a duty primarily performed at the moment by the Coordination Committee.
+Temporary permissions should instead use the "collaborator" feature on Github to
+make it clear that these permissions are temporary in nature.
+
+Additionally, the granter of permissions (usually the Coordination Committee)
+should send a message to the new recipient of write permissions listing the
+responsibilities and expectations that go with this - a template for this email
+is available [in this repo](../messages/maintainer_access.md). That message may
+contain a prompt for a response, which should be cc-ed/forwarded to
+`coordinators[at]astropy.org`.
+
+## Access levels
+
+### Core package maintainers
+
+All maintainers listed for the core package receive *write access* to the
+repository via the **Astropy Core Maintainers** GitHub team.
+
+### Coordinated package maintainers
+
+Coordinated package maintainers receive **admin access** to the coordinated
+package repositories via the **<package name> maintainers** GitHub team (e.g.,
+'astroquery maintainers').
+
+Lower priviledge access (e.g., write, triage) could be assigned to additional
+contributors as separate teams (e.g., 'Astroquery Triage').
+
+### Core package release coordinators
+
+Core package release coordinators receive **admin access** to the core
+repository, as well as the extension-helpers repository
+since releases of those packages may be tightly coupled, as
+well as **write access** to the website repository. This is done via the **Core
+release maintainers** GitHub team.
+
+### Coordination committee
+
+The Coordination Committee members receive **owner access** to
+the astropy GitHub organization. Members who are not familiar or
+comfortable with GitHub administration may opt out. However,
+a majority of the committee should have access. If necessary,
+members should receive GitHub administration training before given access.
+
+In addition, they have access to the project
+credentials (or the shared password manager to access the credentials).
+As with GitHub access above, members may opt out but the majority and training
+rules also apply here.
+
+Regardless of access level, the members are always bound by
```

Comment:
Since the critical sticking point before was whether CoCo members should be the ones with owner's access -- worry about lack of ability, but if not them who? -- and since generally owner access will be needed only rarely and there are risks involved, maybe rewrite this proviso as follows
```
In general, use of the owner access requires permission of the rest of the coordination
committee. Furthermore, regardless of access level, the members are always bound by
[APE 0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst).
For example, the Coordination Committee, let alone just one of its members, cannot
delete or transfer a repository without first obtaining consensus from the community.
```

Reply:
Thanks! I accepted your suggestion and made you co-author of the commit.